src/components/ReportWelcomeText.js

```diff
@@ -62,66 +63,74 @@ const ReportWelcomeText = (props) => {
     );
     const roomWelcomeMessage = ReportUtils.getRoomWelcomeMessage(props.report, props.policies);
     return (
-        <Text style={[styles.mt3, styles.mw100, styles.textAlignCenter]}>
-            {isPolicyExpenseChat && (
-                <>
-                    {/* Add align center style individually because of limited style inheritance in React Native https://reactnative.dev/docs/text#limited-style-inheritance */}
-                    <Text style={styles.textAlignCenter}>
```

Comment:
Is it okay to remove this?

Reply:
Yep! Removing that style shifts the text over to the left, to match the mocks. Not needed anymore.